docs/textbook/Basic_Graphing.ipynb

```diff
@@ -0,0 +1,158 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The easiest way to create a plot begins with writing a function that creates a plot using `matplotlib` and calling `interact` on it. Here we have a simple function that allows the user to control the height of a bar graph with 10 bars."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 51,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "2f0a2526a5e6461a8898a6151b092a1e",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "A Jupyter Widget"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "def plt_bar(y):\n",
+    "    plt.bar(np.arange(10), np.arange(10)+y)\n",
+    "widgets.interact(plt_bar, y=5);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As you might notice, the example above lags and stutters when the slider is dragged around too quickly. This is a limitation of using `matplotlib` and `matplotlib`-based plotting libraries that produce images. The lag occurs because the images are too large to generate and send over the network to match the speed of interaction. This lag can be reduced if you choose to use buttons instead of a slider. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 52,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "546afc549e3c45e0b458f485e4eb2f14",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "A Jupyter Widget"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "widgets.interact(plt_bar, y=widgets.ToggleButtons(options=[10,100,1000]));"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here's another simple example using interact to plot a scatter plot. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 53,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "71b455813c7249f696bd1ae62cfe97c9",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "A Jupyter Widget"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "def plt_scatter(n):\n",
+    "    plt.scatter(np.random.choice(100,n), np.random.choice(100,n))\n",
+    "interact(plt_scatter, n=(0,20));"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In order to handle more complicated visualizations, there are several built-in recommended graphing functions in nbinteract that can be found in the \"nbinteract Graphing\" section. "
```

Comment:
nit:

```
In order to handle more complicated visualizations, we recommend the `nbinteract`
graphing functions described in the "nbinteract Graphing" section.
```

is a bit more succinct. Can we also link to the Graphing section there?